controllers/chat/group/leaveGroupMember.js

```diff
@@ -29,9 +29,9 @@ const leaveGroupMember = async (req, res) => {
       );
       const currentChannel = await client.channel(CHANNEL_TYPE_STRING.GROUP, channelId);
 
-      const textOwnUser = `You leave from this group`;
-      const textTargetUser = `${ownUser.username} has been leave from this group`;
-      const textDefaultUser = `${ownUser.username} has been leave from this group`;
+      const textOwnUser = `You left this group`;
+      const textTargetUser = `${ownUser.username} left this group`;
+      const textDefaultUser = `${ownUser.username} left this group`;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made to the text messages are appropriate and improve the grammar. The new messages are more clear and concise.
```diff
-      const textOwnUser = `You leave from this group`;
-      const textTargetUser = `${ownUser.username} has been leave from this group`;
-      const textDefaultUser = `${ownUser.username} has been leave from this group`;
+      const textOwnUser = `You left this group`;
+      const textTargetUser = `${ownUser.username} left this group`;
+      const textDefaultUser = `${ownUser.username} left this group`;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->